packages/jumpstarter/jumpstarter/common/grpc.py

```diff
@@ -34,19 +35,28 @@ def ssl_channel_credentials(target: str, tls_config):
         return grpc.ssl_channel_credentials()
 
 
-def aio_secure_channel(target: str, credentials: grpc.ChannelCredentials):
+def aio_secure_channel(target: str,
+                       credentials: grpc.ChannelCredentials,
+                       grpc_options: dict[str, Any] | None):
     return grpc.aio.secure_channel(
         target,
         credentials,
-        options=(
-            ("grpc.lb_policy_name", "round_robin"),
-            ("grpc.keepalive_time_ms", 350000),
-            ("grpc.keepalive_timeout_ms", 5000),
-            ("grpc.http2.max_pings_without_data", 5),
-            ("grpc.keepalive_permit_without_calls", 1),
-        ),
+        options=_override_default_grpc_options(grpc_options),
     )
 
+def _override_default_grpc_options(grpc_options: dict[str, Any] | None) -> Sequence[Tuple[str, Any]]:
+    defaults = (
+        ("grpc.lb_policy_name", "round_robin"),
+        # we keep a low keepalive time to avoid idle timeouts on cloud load balancers
+        ("grpc.keepalive_time_ms", 20000),
+        ("grpc.keepalive_timeout_ms", 5000),
+        ("grpc.http2.max_pings_without_data", 5),
+        ("grpc.keepalive_permit_without_calls", 1),
+    )
+    options = dict(defaults, **(grpc_options or {}))
```

Comment:
```suggestion
    options = dict(defaults)
    options.update(grpc_options or {})
```

Reposting the rabbit's suggestion.